Monika-After-Story/MonikaModDev
# Aff system cleanup

Some cleanup + a few new functions for better affection control.

### Testing:
- verify general stability, no aff reset, update script is working
- run unittests
- verify new caps are working
- - also test caps during holidays
- verify `_mas_revertFreshStart`, `_mas_shatterAffection`, `_mas_doFreshStart` are working as intended
- verify gain and lose are randomised with the expected range
- - min gain should be 0+
- verify you can't pass invalid values into the aff functions (like a negative number in aff gain)
- verify new default aff values are good
- - default gain
- - default lose
- - default % lose
- verify bank grants aff and decays as expected
- verify backups are working, try to corrupt affection
- verify apology flag still works as before
- verify spamming compliments doesn't give affection (there's a cooldown)
